tests/testthat/_snaps/sample.md

```diff
@@ -4,23 +4,23 @@
       iris_rec %>% step_sample(size = -1)
     Condition
       Error in `step_sample()`:
-      ! `size` should be a positive number or NULL.
+      ! `size` must be a number larger than or equal to 0 or `NULL`, not the number -1.
```

Comment:
```suggestion
      ! `size` must be `NULL` or a number larger than or equal to 0, not the number -1.
```

with the current wording, i initially thought `NULL` completed some part of the clause "a number larger than or equal to 0"

Reply:
This is handled by `check_number_decimal()` so I'm hesitant to change it